react/Components/titlesArea.tsx

```diff
@@ -17,17 +17,21 @@ const TitlesArea: FC = () => {
       <div className="text-align-center">
         <div className="mb5 t-heading-4">
           {intl.formatMessage(titles.credit)}
-          <b>
-            {intl.formatMessage(titles.money)} {credit}
+          <b className="money">
+            {intl.formatMessage(titles.money)}
+            {credit.toLocaleString('pt-br', { minimumFractionDigits: 2 })}
           </b>
         </div>
         <div className="mb5 t-heading-4">
           {intl.formatMessage(titles.valueGiftCard)}
-          <b>
-            {intl.formatMessage(titles.money)} {valueGiftCard}
+          <b className="money">
+            {intl.formatMessage(titles.money)}{' '}
+            {valueGiftCard.toLocaleString('pt-br', {
+              minimumFractionDigits: 2,
+            })}
```

Comment:
A other way to add space between two words is using `string literals`
```suggestion
            {`${intl.formatMessage(titles.money)}} ${valueGiftCard.toLocaleString('pt-br', {minimumFractionDigits: 2,})}}`
```

Reply:
Sorry for it
Actually this {' '} was added by prettier, I don't need it
But thanks for the warning, I'll remove it.